app/gui2/src/components/visualizations/ScatterplotVisualization.vue

```diff
@@ -178,6 +185,10 @@ const focus = ref<Focus>()
 const shouldAnimate = ref(false)
 const xDomain = ref([0, 1])
 const yDomain = ref([0, 1])
+const selectionEnabled = ref(false)
+const enableSelection = () => {
```

Comment:
- The name says `enable`, but the implementation toggles.
- I don't see a need to wrap this in a function. It's a one liner, and is unlikely to need future changes--if any logic needs to be run when the value is changed, the Vue solution is to use watchers.